toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -32,7 +32,7 @@ var (
 	DefaultMkfsOptions = map[string][]string{
 		"ext2": {"-b", "4096", "-O", "none,sparse_super,large_file,filetype,resize_inode,dir_index,ext_attr"},
 		"ext3": {"-b", "4096", "-O", "none,sparse_super,large_file,filetype,resize_inode,dir_index,ext_attr,has_journal"},
-		"ext4": {"-b", "4096", "-O", "none,sparse_super,large_file,filetype,resize_inode,dir_index,ext_attr,has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize"},
+		"ext4": {"-b", "4096", "-O", "none,sparse_super,large_file,filetype,resize_inode,dir_index,ext_attr,has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize,^metadata_csum_seed"},
```

Comment:
I would suggest, adding a comment that `^metadata_csum_seed` so that the intent is clear for someone who would be glancing the file. It also reduces back-and-forth to double check if this might be a typo :)